pom.xml

```diff
@@ -49,6 +49,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-jaxp</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-health</artifactId>
+        </dependency>
```

Comment:
Hmmm. Does it really bring something to just have this extension around? OpenShift will behave differently?

Reply:
Yes, it exposes `/q/health` and should also fix the warning shown in the console:

![image](https://user-images.githubusercontent.com/54133/114275228-206ebf00-99f8-11eb-95de-cd09a2b3e227.png)